src/manifolds/VectorBundle.jl

```diff
@@ -635,6 +635,12 @@ function inner(B::VectorBundle, p, X, Y)
     VYM, VYF = submanifold_components(B.manifold, Y)
     return inner(B.manifold, px, VXM, VYM) + inner(B.fiber, Vx, VXF, VYF)
```

Comment:
```suggestion
    return inner(B.manifold, px, VXM, VYM) + inner(VectorSpaceAtPoint(B.fiber, px), Vx, VXF, VYF)
```

It was wrong here. For arbitrary vector bundles it should be this way I think. `Vx` is a point on the vector space manifold and `VXF` and `VYF` are tangent vectors to that vector space at point.